### PR TITLE
Adds repeat keyword to levels 11+

### PR DESCRIPTION
**Pull request by huang-v**
**Description**

Adds ability to use the keyword "repeat" in levels 11+

**Fixes _issue or discussion number_**

#1942

**How to test**

Repeat tests were added to level 11 as that is where the keyword is removed from the cumulative command variable and all levels were manually tested 

**Checklist**
  
- [x] Describes changes in the format above (present tense)
- [x] Links to an existing issue or discussion 
- [x] Has a "How to test" section

### CODE REVIEW

File: hedy.py [hunk 1]
```diff
@@ -115,14 +115,14 @@ def promote_types(types, rules):
     8 :['ask', 'is', 'print', 'forward', 'turn', 'sleep', 'at', 'random', 'add', 'to', 'remove', 'from', 'in', 'if', 'else', 'repeat', 'times'],
     9 :['ask', 'is', 'print', 'forward', 'turn', 'sleep', 'at', 'random', 'add', 'to', 'remove', 'from', 'in', 'if', 'else', 'repeat', 'times'],
     10 :['ask', 'is', 'print', 'forward', 'turn', 'sleep', 'at', 'random', 'add', 'to', 'remove', 'from', 'in', 'if', 'else', 'repeat', 'times', 'for'],
-    11 :['ask', 'is', 'print', 'sleep', 'at', 'random', 'add', 'to', 'remove', 'from', 'in', 'if', 'else', 'for', 'range'],
```
Comment (Felienne) [inline hedy.py:118]: It seems we are adding and not removing repeat here?

**Review by Felienne — CHANGES_REQUESTED** [review]
Hi @huang-v,

This PR seems to be a mix of adding and removing repeat? The title of the PR says adding but the text says removing, and the code seems to mix it too.

I know that the issue itself is also formulated very vaguely, so maybe this is our bad also!

Ultimately we want to have smooth level borders and allow repeat that way, so maybe we should just leave this all as it is now.

**Comment by huang-v** [comment]
Sorry for the confusing wording.
I was told that "The idea was to add repeat to levels over 11."

The list of available keywords is based on the command variable in the lark files. Before, the repeat keyword was removed at level 11. I removed this removal however which essentially adds repeat to levels 11+

**Comment by Felienne** [comment]
ok got it!

**Pull request merged**